src/Pages/Product/components/ProductReview/ProductReview.scss

```
@@ -0,0 +1,81 @@
+@import '../../../../Styles/variables';
+
+.productReview {
+  .reviewOutbox {
+    .reviewTitle {
+      display: flex;
+      border-bottom: 1px solid $lightGrey;
+      padding-bottom: 10px;
+      .title {
+        margin-right: 10px;
+        font-weight: 800;
+      }
+      .countReview {
+        font-weight: 600;
+      }
+    }
+    .reviewContentsOutbox {
+      .starRate {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        margin: 20px;
+        width: 650px;
+        height: 100px;
+        // border: 1px solid $lightGrey;
+        border-radius: 30px;
+        box-shadow: 6px 2px 20px -6px $lightGrey;
+        .stars {
+          img {
+            width: 20px;
+          }
+        }
+        .reviewPoint {
+          margin-left: 60px;
+          line-height: $product-description-line-height;
+        }
+      }
+    }
+    .commentLists {
+      .commentContent {
+        border-bottom: 1px solid $lightGrey;
+        border-width: thin;
+        padding: 30px 0;
+        .commentHeading {
+          display: flex;
+          position: relative;
+          color: $lightGrey;
```

```suggestion
    .commentLists {
      .commentContent {
        border-bottom: 1px solid $lightGrey;
        border-width: thin;
        padding: 30px 0;
        
        .commentHeading {
          display: flex;
          position: relative;
          color: $lightGrey;
```
가독성을 위해 새로운 셀렉터 앞에서는 한줄 띄워주세요

---

네엡!!!